src/test/java/org/folio/edge/orders/utils/OrdersMockOkapi.java

```diff
@@ -39,6 +39,7 @@ public OrdersMockOkapi(int port, List<String> knownTenants) throws IOException {
   public Router defineRoutes() {
     Router router = super.defineRoutes();
     router.route(HttpMethod.GET, "/gobi/validate").handler(this::validateHandler);
+    router.route(HttpMethod.POST, "/gobi/validate").handler(this::postValidateHandler);
```

Comment:
You can probably reuse the existing validateHandler here and just check the request's HTTP method (see earlier comments)

Reply:
Done

Reply:
I don't see the changes committed

Reply:
If you click the view changes, it will still take you to the old code. If you go to the files changed tab directly, the new changes will be reflected